symplyphysics/laws/electricity/circuits/waveguides/specific_conductivity_of_coaxial_waveguide.py

```diff
@@ -2,13 +2,18 @@
 Specific conductivity of coaxial waveguide
 ==========================================
 
-A coaxial waveguide is an electrical cable consisting of a central conductor and a shield arranged coaxially and separated
-by an insulating material or an air gap. It is used to transmit radio frequency electrical signals.
-The specific conductivity of a coaxial waveguide depends on the frequency of signal and the specific capacitance of coaxial waveguide,
-as well as on the tangent of the dielectric loss angle of the insulator material.
-In general, the tangent of the dielectric loss angle is a characteristic of the material. The angle itself is usually not searched for or indicated.
-The tangent of this angle is found as the ratio of the active current to the reactive current. The value of this value for each material is in the tables
-in the public domain.
+A coaxial waveguide is an electrical cable consisting of a central conductor and a
+shield arranged coaxially and separated by an insulating material or an air gap. It is
+used to transmit radio frequency electrical signals. The specific conductivity of a
+coaxial waveguide depends on the frequency of signal and the specific capacitance of
+coaxial waveguide, as well as on the tangent of the dielectric loss angle of the
+insulator material.
+
+**Notes:**
+    
+#. The tangent of the dielectric loss angle is found as the ratio of the active current
```

Comment:
You can remove this note. We have a symbol for this so all relevant description is in there.